README.md

```diff
@@ -3,12 +3,14 @@ About readline
 
 Home: https://cnswww.cns.cwru.edu/php/chet/readline/rltop.html
 
-Package license: GPL3
+Package license: GPL-3.0
 
 Feedstock license: BSD 3-Clause
 
 Summary: library for editing command lines as they are typed in
 
+The standard Python readline extension statically linked against the GNU
```

Comment:
Here as well. I wonder if this was auto generated upstream or manually and if manually why?

Reply:
Auto-generated and the re-render below should take care of it.